scripts/triggers-end-to-end-tests/run.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+source scripts/set-default-credentials.sh
+
+npm link
+
+mocha \
+  --require ts-node/register \
+  --require source-map-support/register \
+  --require src/test/helpers/mocha-bootstrap.js \
+  --exit \
+  scripts/triggers-end-to-end-tests/tests.ts
```

Comment:
Nit: use a relative path to `test.ts`?

Reply:
This is relative to where the script is run (which is the root directory - and matches the other `run.sh` files)